src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1115,6 +1115,12 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 			item.eatResultItem = new MItemBlock(modElement.getWorkspace(),
 					emptyLists ? "" : getRandomMCItem(random, blocksAndItems).getName());
 			item.onFinishUsingItem = new Procedure("procedure3");
+			item.enableRanged = _true;
+			item.projectile = new ProjectileEntry(modElement.getWorkspace(),
+					getRandomDataListEntry(random, ElementUtil.loadArrowProjectiles(modElement.getWorkspace())));
+			item.shootConstantly = _true;

```

Comment:
You use same condition to enable ranged and to shoot constantly, meaning it will always result in testing ranged item that shoots constantly. I would use eg. emptyLists here to make sure both combinations are tested